pos_pricelist/models/pos_config.py

```diff
@@ -4,15 +4,11 @@
 
 from odoo import _, api, fields, models
 from odoo.exceptions import ValidationError
+from oca.decorators import foreach
```

Comment:
Possibly you need to [disable the pylint message for this line](https://travis-ci.org/OCA/pos/jobs/350388592#L734) because it's handled in a higher level.

Odoo is so stupid that it imports addons when not installed without checking if there's an exception when doing so :man_facepalming: 

Another option would be to:

```python
try:
    from oca.decorators import foreach
except ImportError:
    def foreach():
        return api.one
```

But that's uglier IMHO.

Reply:
There's a better solution proposed by @hbrunn in another issue opened in the library. I don't like too much to have 3 issues/PRs for the same thing.